src/AzureDeprecation.Notices.Management/MessageHandlers/NewAzureDeprecationV1MessageMessageHandler.cs

```diff
@@ -84,19 +80,22 @@ private List<string> DetermineRequiredLabels(NewAzureDeprecationV1Message newNot
                 "verified"
             };
 
-            foreach (var service in newNoticeV1MessageQueueMessage.Impact.Services)
-            {
-                var label = LabelFactory.GetForService(service);
-                labels.Add(label);
+            if (newNoticeV1MessageQueueMessage.Impact is not null)
```

Comment:
If it's not there, we should add "unknown" labels

Reply:
Can you please check if I got what you asked for?

Reply:
The idea is to rather use `services:unknown`, `area:unknown`, etc - Added suggestion below